### DISCUSSION TITLE
Retrieving Lighthouse User Flow Result as JSON

### DISCUSSION THREAD

**Discussion by alicehaupais** [GoogleChrome/lighthouse#14551] [Q&A] [answered]
Hello,

I would like to implement a Lighthouse Analysis after running a user flow on a page.
I check the User Flow functionality : https://web.dev/lighthouse-user-flows/ and it would be exactly what I want to implement.
Is there a way to retrieve the outputs as JSON instead of an HTML Report ?

It was the case using the npm package lighthouse  by setting it as an option : 
 const options = {  output: 'json' }

Thank you in advance for your help

**Comment by alicehaupais** [comment, marked as answer]
Hello,

I was able to find the command to perform it : 
`  const lighthouseResults = await flow.createFlowResult()`

**Reply by adamraine** [reply]
Yup this is the correct way to do it
